README.md

```diff
@@ -0,0 +1,48 @@
+stacker_blueprints
+==================
+
+[![circleci](https://circleci.com/gh/cloudtools/stacker_blueprints.svg?style=shield)](https://circleci.com/gh/cloudtools/stacker_blueprints)
+[![pypi package](https://badge.fury.io/py/stacker_blueprints.svg)](https://badge.fury.io/py/stacker_blueprints)
+[![slack](https://empire-slack.herokuapp.com/badge.svg)](https://empire-slack.herokuapp.com)
+
+An attempt at a common Blueprint library for use with `stacker <https://github.com/cloudtools/stacker>`_.
+
+If you're new to stacker you may use `stacker_cookiecutter <https://github.com/cloudtools/stacker_cookiecutter>`_ to setup your project.
+
+# Quick start
+
+>NOTES: it will create aws resources and generate cost on your aws account. `make destroy` them all if not required any more.
+
+Make sure you have permission to access aws, review below files if you need change anything.
+
+1) Adjust namespace to a global unique name, the name in example will be used in s3 bucket name, and has been used. 
+2) I have issue to start with pre_build task, if you get same problem, comment that session. 
```

Comment:
We should probably just fix the pre_build task if there is an issue- can you share what problem you're seeing as an issue?